src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -131,9 +131,19 @@ public sealed partial class EnumBuilder : System.Reflection.TypeInfo
         public override System.Type? GetInterface(string name, bool ignoreCase) { throw null; }
         public override System.Reflection.InterfaceMapping GetInterfaceMap(System.Type interfaceType) { throw null; }
         public override System.Type[] GetInterfaces() { throw null; }
-        [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.All)]
+        [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicFields | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicFields |
+            System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicMethods | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicMethods |
```

Comment:
These need to be all in one line, or else the next person who generates the ref file will need to fix them.

We shouldn't manually fix up the generated code. We should leave it as the generator generates it.

See [https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md)